### PR TITLE
Java integration tests

### PR DESCRIPTION
**Pull request by tyrcho**
I finally managed to make a complete implementation of the gazr spec for a java project ! As I am a spring-boot newbie this was more challenging than I expected ...

@tsunammis what do you think about this approach in the README with checkboxes ? And of the `make all` (used to check in one step the gazr implementation) ?

### CODE REVIEW

**Comment by tyrcho** [comment]
closes #3

File: examples/java_springboot/Makefile [hunk 1]
```diff
@@ -1,47 +1,38 @@
-.PHONY: run watch test-unit test-functional test format style complexity
+.PHONY: run watch test-unit test-functional test-integration test format style complexityall prepare
```
Comment (tsunammis) [inline examples/java_springboot/Makefile]: missing space between `complexity` and `all`

**Review by tsunammis — CHANGES_REQUESTED** [review]

**Comment by tsunammis** [comment]
@tyrcho Really good example. I should refactor the python examples based on this one. 👍 

Regarding the `all` task, I'm thinking about it because, in your example, you don't execute the whole list of Gazr's tasks _(`watch` is missing)_, and this is logical _(conflict between `run` and `watch`)_. 

So, I think that you can let it in this example if it seems straightforward from your point of view.

File: examples/java_springboot/Makefile (hunk 1 above)
Comment (tyrcho) [reply examples/java_springboot/Makefile]: good catch !

**Review by tsunammis — APPROVED** [review]

**Pull request merged**